addons/account/static/src/components/duplicate_bill_warninng/duplicate_bill_warning.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="account.DuplicateBillWarning">
+        <t t-if="duplicate_bills_count > 0">
+            <div class="alert alert-warning mb-0 w-100" role="alert">
+                Warning: this bill might be a duplicate of
```

Comment:
If we want to make the field more generic, this could go in the form view instead of in the widget.
Also, all the variable names could get rid of `bill` or `duplicate`. Just thing of it as a clickable `many2many_tags`

Reply:
aright !